src/VisualStudio/Core/Def/Packaging/PackageInstallerServiceFactory.cs

```diff
@@ -570,5 +576,85 @@ public uint GetTokens(uint dwMaxTokens, IVsSearchToken[] rgpSearchTokens)
                 return 0;
             }
         }
+
+        private class PackageInstallerServicesProxy : IPackageInstallerServicesProxy
```

Comment:
You could also have just one interface and class on our side, since you use all of these in the exact same place.